readme.md

```diff
@@ -43,3 +43,5 @@ update tests for new email sending flow
 use new logo
 
 update auth flow
+
+more documentation
```

Comment:
any input here @botany-demo @botany-dummy

Reply:
maybe add more of the documentation?

Reply:
cool, good idea

Reply:
ok, added it @botany-lead

Reply:
perfect, looks good to me